homeassistant/components/qnap/const.py

```diff
@@ -0,0 +1,210 @@
+"""The Qnap constants."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorStateClass,
+    SensorEntityDescription,
+)
+from homeassistant.const import (
+    DATA_GIBIBYTES,
+    DATA_RATE_MEBIBYTES_PER_SECOND,
+    PERCENTAGE,
+    UnitOfTemperature,
+)
+
+ATTR_DRIVE = "Drive"
+ATTR_ENABLED = "Sensor Enabled"
+ATTR_IP = "IP Address"
+ATTR_MAC = "MAC Address"
+ATTR_MASK = "Mask"
+ATTR_MAX_SPEED = "Max Speed"
+ATTR_MEMORY_SIZE = "Memory Size"
+ATTR_MODEL = "Model"
+ATTR_PACKETS_TX = "Packets (TX)"
+ATTR_PACKETS_RX = "Packets (RX)"
+ATTR_PACKETS_ERR = "Packets (Err)"
+ATTR_SERIAL = "Serial #"
+ATTR_TYPE = "Type"
+ATTR_UPTIME = "Uptime"
+ATTR_VOLUME_SIZE = "Volume Size"
+
+PLATFORMS = ["sensor"]
+DEFAULT_NAME = "QNAP"
+DEFAULT_PORT = 8080
+DEFAULT_TIMEOUT = 5
+DOMAIN = "qnap"
+
+NOTIFICATION_ID = "qnap_notification"
+NOTIFICATION_TITLE = "QNAP Sensor Setup"
+VOLUME_NAME = "volume"
+
+
+@dataclass
+class QNapSensorEntityDescription(SensorEntityDescription):
+    """Represents an Flow Sensor."""
+
+    stype: str | None = None
+
+
+SENSOR_TYPES: tuple[QNapSensorEntityDescription, ...] = (
```

Comment:
This constant is better kept in `sensor.py`

Reply:
Moved